RecordingApp/app/src/main/java/wycliffeassociates/recordingapp/ProjectManager/dialogs/ProjectInfoDialog.java

```diff
@@ -77,9 +84,9 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
         projectTitle.setText(book + " (" + bookCode + ")");
         languageTitle.setText(language + " (" + languageCode + ")");
         translator.setText(translators);
-        if(translation.compareTo("ulb") == 0) {
+        if (translation.compareTo("ulb") == 0) {
```

Comment:
So, my reason for using compare to is that I know it works for string equality. 
equals() is a method inherited from object, and my expectation was that it compared the String object references, rather than testing for string equality.

This does check for string equality, and future uses should use .equals()

So yeah, we should fix existing uses as we find them. Thanks.